packages/angular_devkit/build_angular/BUILD.bazel

```diff
@@ -217,22 +212,23 @@ jasmine_node_test(
     srcs = [":build_angular_test_lib"],
 )
 
-# @external_begin
```

Comment:
Guessing we to keep the @external_begin & end here as well to not break google3 when this is synced in

Reply:
Nevermind. Not needed as per https://github.com/angular/angular-cli/pull/22258#discussion_r759582051